compatibility/common/src/main/java/org/projectnessie/tools/compatibility/internal/ServerKey.java

```diff
@@ -52,19 +56,39 @@ public static ServerKey forContext(
             instance ->
                 findRepeatableAnnotations(instance, NessieServerProperty.class)
                     .forEach(prop -> config.put(prop.name(), prop.value())));
-    return new ServerKey(version, databaseAdapterName, config);
+
+    String storeKindStr = config.get(STORAGE_KIND_PROPERTY);
+    StorageKind kind = StorageKind.DATABASE_ADAPTER;
+    if (storeKindStr != null) {
+      kind = StorageKind.valueOf(storeKindStr);
+    }
+
+    switch (kind) {
+      case DATABASE_ADAPTER:
+        return new ServerKey(version, databaseAdapterName, kind, config);
+
+      case OBJECT:
```

Comment:
Hm - why not `PERSIST` as everywhere else?

Reply:
sure, "object" relates to `ObjId` in `Persist`, but `PERSIST` LGTM too.

Reply:
renamed